Mathlib/Topology/Algebra/InfiniteSum/Group.lean

```diff
@@ -126,6 +126,26 @@ theorem hasProd_ite_div_hasProd [DecidableEq β] (hf : HasProd f a) (b : β) :
     rw [Function.update_apply]
   · rw [div_mul_eq_mul_div, one_mul]
 
+/-- A more general version of `Multipliable.congr`. -/
+@[to_additive "A more general version of `Summable.congr`"]
+theorem Multipliable.congr_cofinite (hf : Multipliable f) (hfg :f =ᶠ[cofinite] g) :
```

Comment:
Can we put these ones closer to `Multipliable.congr`?

Reply:
No since the proof need results proved just above in this file.